pkg/cloud/cloud.go

```diff
@@ -521,7 +523,8 @@ func (c *cloud) waitForVolume(ctx context.Context, volumeID string) error {
 			case "available":
 				return true, nil
 			case "creating":
-				return false, fmt.Errorf("volume %s is still being created", volumeID)
+				glog.Warningf("Volume %s is still being created, will retry", volumeID)
```

Comment:
Nit: change `will retry` to `waiting` or similar because it's not going to retry the CreateVolume operation.